resqpy/organize/earth_model_interpretation.py

```diff
@@ -0,0 +1,121 @@
+from .organize_functions import (equivalent_extra_metadata, extract_has_occurred_during, equivalent_chrono_pairs,
```

Comment:
I don't think the brackets are needed here.

Reply:
Think brackets are indeed needed if list of objects to import runs on multiple lines, as in this case